app/src/main/java/io/github/droidkaigi/confsched/activity/MainActivity.java

```diff
@@ -85,7 +95,7 @@ protected void onDestroy() {
     private void initView() {
         setSupportActionBar(binding.toolbar);
         ActionBarDrawerToggle toggle = new ActionBarDrawerToggle(this,
-                binding.drawer, binding.toolbar, R.string.open, R.string.close);
+                                                                        binding.drawer, binding.toolbar, R.string.open, R.string.close);
```

Comment:
Sorry, I have to work on my Formatting Seriously...and of course need to look at diff. 
I fixed my Code Style, so it wont happen next time.